Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationBroadcaster.cs

```diff
@@ -16,11 +16,12 @@ namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
     /// </summary>
     public class StateSynchronizationBroadcaster : Singleton<StateSynchronizationBroadcaster>
```

Comment:
>Singleton<StateSynchronizationBroadcaster> [](start = 51, length = 42)

Would be nice to have a base class CommandService<T> : Singleton<T> to contain the shared implementation of adding and removing handlers and broadcasting a command to the right ones